tests/test_jira.py

```diff
@@ -199,6 +273,24 @@ def test_fetch_empty(self):
         self.assertRegex(request.path, '/rest/api/2/search')
         self.assertDictEqual(request.querystring, expected_req)
 
+    @httpretty.activate
+    def test_filter_custom_fields(self):
+        """Test that all the fields returned are just custom"""
+        body = read_file('data/jira/jira_fields.json')
+
+        httpretty.register_uri(httpretty.GET,
+                               JIRA_FIELDS_URL,
+                               body=body, status=200)
+
+        jira = Jira(JIRA_SERVER_URL)
```

Comment:
You're testing a static method. You don't need to instantiate the class. You just need to call the method this way:
```
Jira.filter_custom_fields(json)
```